server/api/db/sqldb/db.py

```diff
@@ -1741,6 +1741,66 @@ def update_function(
             self._upsert(session, [function])
             return function.struct
 
+    def update_function_external_invocation_url(
+        self,
+        session,
+        name,
+        external_invocation_url: str,
+        project="",
+        tag="",
+        hash_key="",
+        operation="add",
+    ):
+        project = project or config.default_project
+        normalized_function_name = mlrun.utils.normalize_name(name)
+        query = self._query(
+            session, Function, name=normalized_function_name, project=project
+        )
+        uid = self._get_function_uid(
+            session=session,
+            name=normalized_function_name,
+            tag=tag or "latest",
+            hash_key=hash_key,
+            project=project,
+        )
+        if uid:
+            query = query.filter(Function.uid == uid)
+        function = query.one_or_none()
+        if not function:
+            logger.debug(
+                "Function is not found",
```

Comment:
"Not found" can be in too many flows
```suggestion
                "Function is not found, skipping external invocation urls update",
```